modules/ROOT/pages/maven-reference.adoc

```diff
@@ -77,6 +77,73 @@ When you create a new Mule project in Anypoint Studio, the required Mule reposit
 </pluginRepository>
 ----
 
+== MuleSoft Enterprise Repository
```

Comment:
```suggestion
```
See above. We're still introducing the three tasks or requirements for access to the enterprise repos.